common/src/main/scala/uk/ac/wellcome/models/IdentityState.scala

```diff
@@ -0,0 +1,23 @@
+package uk.ac.wellcome.models
+
+sealed trait IdentityState[+T]
+
+sealed trait IdentifieableOrUnidentifiable[+T] extends IdentityState[T]
```

Comment:
As per our discussion last week, I think this trait should definiately be renamed to something like `Displayable`.